Library/Formula/keybase.rb

```diff
@@ -1,33 +1,28 @@
 class Keybase < Formula
   desc "Command-line interface to Keybase.io"
   homepage "https://keybase.io/"
-  url "https://github.com/keybase/node-client/archive/v0.8.23.tar.gz"
-  sha256 "20e7e7e7a125ddb3b0e7023c3dcaf12f12c7b183822592b164ca11dd1e6f9a25"
-  head "https://github.com/keybase/node-client.git"
+  url "https://github.com/keybase/client/archive/v1.0.0-47.tar.gz"
+  sha256 "a026fa0df54909989a50d607c0ac942857aa4ca7de00d5adef8ef30f403a082d"
+  version "1.0.0-47"
+
+  head "https://github.com/keybase/client.git"
 
   bottle :unneeded
```

Comment:
Ok.  We weren't insinuating anything about your CI security.  Sorry if that was the message you received.

In an ideal world, it would be nice to be able to sign bottles.